contracts/BoosterFeeCollector.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.3;
+
+import "./interfaces/IPlatypusVoter.sol";
+import "./interfaces/IBoosterFeeCollector.sol";
+import "./lib/Ownable.sol";
+import "./lib/ERC20.sol";
+import "./lib/SafeERC20.sol";
+import "./lib/SafeMath.sol";
+
+contract BoosterFeeCollector is Ownable, IBoosterFeeCollector {
+    using SafeMath for uint256;
+    using SafeERC20 for IERC20;
+
+    uint256 internal constant BIPS_DIVISOR = 10000;
+
+    IERC20 public constant PTP = IERC20(0x22d4002028f537599bE9f666d1c4Fa138522f9c8);
+    IPlatypusVoter public constant VOTER = IPlatypusVoter(0x40089e90156Fc6F994cc0eC86dbe84634A1C156F);
+
+    mapping(address => uint256) public boostFeeBips;
+    address public boosterFeeReceiver;
+    bool paused = false;
+
+    constructor(address _boosterFeeReceiver) {
+        boosterFeeReceiver = _boosterFeeReceiver;
+    }
+
+    function setBoostFee(address _strategy, uint256 _boostFeeBips) external override onlyOwner {
+        boostFeeBips[_strategy] = _boostFeeBips;
+    }
+
+    function setBoosterFeeReceiver(address _boosterFeeReceiver) external override onlyOwner {
+        boosterFeeReceiver = _boosterFeeReceiver;
```

Comment:
emit BoostFeeReceiverUpdated(new, old)

Reply:
added, chose (old, new) to keep it consistent with order in YakStrategy